README.md

```diff
@@ -0,0 +1,127 @@
+# UI Kit
+Components and tools for building graphical UIs for Agoric dapps
+
+## Setup
+
+These packages require SES ([learn more](https://github.com/endojs/endo/tree/master/packages/ses)).
+See this [example](https://github.com/Agoric/dapp-inter/blob/main/src/main.tsx#L1) for enabling SES in your application.
+Setup may vary by environment.
+
+## Reading Contract Data (vstorage)
+
+`makeAgoricChainStorageWatcher` can be used to subscribe to updates to vstorage.
+It polls the RPC periodically, automatically batching and de-duping requests for efficiency.
+
+See https://github.com/p2p-org/p2p-agoric-vstorage-viewer to explore vstorage more.
+
+*package.json:*
+```
+"@agoric/rpc": "^0.4.1-dev-7cf64bb.0"
+```
+
+
+*app.ts:*
+```ts
+import {
+  makeAgoricChainStorageWatcher,
+  AgoricChainStoragePathKind as Kind
+} from '@agoric/rpc';
+
+const watcher = makeAgoricChainStorageWatcher(rpc, chainName);
```

Comment:
I'm used to passing network access (`fetch`) in explicitly. I don't expect to change that here, but it's worth indexing under https://github.com/Agoric/agoric-sdk/issues/2160